kimchi/src/circuits/expr.rs

```diff
@@ -3075,7 +3079,7 @@ where
                 });
                 res
             }
-            Atom(Cell(v)) => v.latex(),
+            Atom(Cell(v)) => v.latex(&mut HashMap::new()),
```

Comment:
why isn't the cache in the arg of the function given as arg to the recursive call ?

Reply:
Not the same type, and also, it is unused in any case.